### PR TITLE
Adding 2.8.0 gwt-module.dtd doctype.

### PR DESCRIPTION
**Pull request by branflake2267**
[Review on Reviewable](https://reviewable.io/reviews/gwtproject/gwt-site/155)

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -22,6 +22,9 @@
       <resource>
         <directory>src/main/site</directory>
       </resource>
+      <resource>
```
Comment (branflake2267) [inline pom.xml]: Eclipses uses this, and configures it as a resource directory, which makes it easier to navigate.

File: src/main/site/doctype/2.8.0/gwt-module.dtd [hunk 2]
```diff
@@ -0,0 +1,206 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (branflake2267) [inline src/main/site/doctype/2.8.0/gwt-module.dtd]: Copied from master

**Comment by manolo** [comment]
LGTM

**Comment by tbroyer** [comment]
FWIW, the DTD is out of date (in 2.7 too): https://groups.google.com/d/msg/google-web-toolkit/Cw417h8IakY/NAqpX3f0DgAJ

**Comment by branflake2267** [comment]
@tbroyer I copied the 2.7.0 dtd and pulled it against master and I'm only seeing white space difference. 
https://github.com/gwtproject/gwt-site/compare/master...branflake2267:patch-1 

https://gwt.googlesource.com/gwt.git/+/2.7.0/distro-source/core/src/gwt-module.dtd

**Comment by tbroyer** [comment]
That's what I'm saying: the DTD is not in sync with the actual "schema" as understood by the code.

**Pull request merged**